src/utils/format/formatTime.ts

```diff
@@ -141,3 +141,21 @@ export const formatTime = (timestamp: BigNumber | undefined) => {
   const timeWeekdayString = timeDate.toLocaleString('en-US', formatOptions)
   return isoDateString + ', ' + timeWeekdayString
 }
+
+export const formatTime = (timestamp: bigint | undefined) => {
```

Comment:
I feel like if this is the exact same as the other one, we should either just nuke the `formatTimeBN` and replace uses with the new bigint fn.

`formatTime(BigInt(myBigInt.toString()))`

In fact, maybe we should have a function called `toBigInt`

```js
function toBigInt(bn: BigNumber | undefined) {
    return bn ? BigInt(bn.toString()) : undefined
}
```

So then the pattern for 'migrating' to bigint becomes:
- identify hook or utility that could be bigint'd
- rewrite it with bigint, accept bigint params etc
- refactor existing usages to transform bignumber to bigint with `toBigInt`.

WDYT?

Reply:
Good call, done for this instance